DEPS

```diff
@@ -606,29 +606,51 @@ deps = {
     'packages': [
       {
         'package': 'fuchsia/third_party/clang/mac-amd64',
-        'version': 'NyopqdQvXY11sckCZIz7FdXLgQZep1ssg1IYBX09ZPwC'
+        'version': 'git_revision:3a20597776a5d2920e511d81653b4d2b6ca0c855'
```

Comment:
Is this going to confuse or get clobbered by the toolchain autorollers? E.g. https://autoroll.skia.org/r/clang-mac-flutter-engine

Reply:
Also, shouldn't we still be tracking the toolchain that the Fuchsia toolchain team is officially releasing?

Reply:
Revisions instead of instance ids make it easier to see whether the platforms are in sync and make it easier to match things back to the source repositories. Dart's DEPS and Fuchsia's manifest both use revisions instance of instance ids. This is the revision currently used by Fuchsia integration.

Reply:
If changes aren't also made to the autoroller, then the next autoroller roll will clobber these back to instance ids (if it can understand the new layout at all.)